build/commands/lib/config.js

```diff
@@ -625,8 +625,17 @@ Config.prototype.buildArgs = function () {
 
     args.android_aab_to_apk = this.androidAabToApk
 
-    // This optimization causes crash on Android 8 and Android 8.1 arm64 devices.
-    args.use_relr_relocations = false
+    if (this.targetArch == "arm64") {
+      // Flag use_relr_relocations is incompatible with Android 8 arm64, but
+      // makes huge optimizations on Android 9 and above.
+      // Decision is to specify android:minSdkVersion=28 for arm64 and keep
+      // 26(default) for arm32.
+      // Then:
+      //   - for Android 8 and 8.1 GP will supply arm32 bundle;
+      //   - for Android 9 and above GP will supply arm64 and we can enable all
+      //     optimizations.
+      args.default_min_sdk_version = 28
```

Comment:
@AlexeyBarabash I also wonder how it work for Chrome.
There is also `min_sdk_version` param and it's 29 for [system webview](https://github.com/brave/chromium/blob/main/android_webview/system_webview_apk_tmpl.gni#L119-L121)

Reply:
@atuchin-m from what I see, branded Google Chrome  has `min_sdk` is `26`.
And it doesn't crash because it uses 32-bit libs on Android 8 - can be checked through about:version and through 
`android:versionCode=616716421`,  

[last digit 1 means](https://source.chromium.org/chromium/chromium/src/+/main:build/util/android_chrome_version.py;l=217;drc=f4a00cc248dd2dc8ec8759fb51620d47b5114090;bpv=0?q=build%2Futil%2Fandroid_chrome_version.py&ss=chromium%2Fchromium%2Fsrc)  32-bit libs are used and 64-bit webview libs.

I don't know how Google made Google Play to ship arm32 on Android 8 devices.